src/sources/aws_kinesis_firehose/mod.rs

```diff
@@ -40,6 +40,8 @@ pub struct AwsKinesisFirehoseConfig {
     #[configurable(metadata(docs::examples = "A94A8FE5CCB19BA61C4C08"))]
     access_key: Option<SensitiveString>,
```

Comment:
We should deprecate this option in favor of the new one. See https://github.com/vectordotdev/vector/blob/master/docs/DEPRECATION.md for the process.